tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
 
 [testenv]
 commands =
     python setup.py dev
-    python setup.py test -q # -q does not suppress errors
-
-[testenv:cover]
-commands =
-    python setup.py dev
-    python setup.py nosetests --with-coverage --cover-min-percentage=47
+    py.test --cov letsencrypt
```

Comment:
I did only a quick search for it, but didn't find it.
Guess it would be rather pytest-cov that would have something like that.